ci/spack.yaml

```diff
@@ -0,0 +1,22 @@
+spack:
```

Comment:
Here we are using a spack environment to set up a build for all dependencies (except for veloc)